src/server_manager/ui_components/app-root.html

```diff
@@ -42,8 +43,20 @@
     <style>
       :host {
         width: 100%;
-        --primary-green: #00bfa5;
-        --light-gray: color: rgba(0,0,0,0.54);
+        --status-bar-width: 48px;
```

Comment:
+1 for variable names. I think we can improve this one - throughout this file I see several names for the sidebar and it changes when it's minimised/maximised. I think we should settle on names for the bar and its two states - it could be "sidebar" along with "narrow" and "wide" (or empty string, to indicate that that's the normal state).

WDYT?

Reply:
Yep, that makes sense though I found it a little surprising - I would have assumed the drawer widget itself supported a minimised state.